rdkit/sping/util/HTMLPiddler.py

```diff
@@ -195,7 +197,7 @@ def send_line_break(self):
     self.x = self.indent
     self.atbreak = 0
     if TRACE:
-      raw_input('lb')
+      input('lb')
```

Comment:
I don't think it matters here, but I don't think `input()` in python2 is really the same as `raw_input()` in python2.